docker-compose.yml

```diff
@@ -74,6 +71,35 @@ services:
     networks:
       - bykstack
 
+  tim:
```

Comment:
TIM part in docker-compose.yml should not be present, as in dev we run only one TIM separately from modules and all the modules use it.